default/python/AI/AIstate.py

```diff
@@ -368,6 +366,13 @@ def update_system_status(self):
         min_hidden_attack = 4
         min_hidden_health = 8
 
+        # TODO: Variables that are recalculated each turn from scratch should not be stored in AIstate
```

Comment:
`should be stored in turn_state.State`

https://github.com/freeorion/freeorion/blob/master/default/python/AI/turn_state.py#L10